src/App/Settlements/SettlementDetailPositions/connectors.ts

```diff
@@ -1,25 +0,0 @@
-import { connect, ConnectedProps } from 'react-redux';
```

Comment:
Is the details portion just being removed?

Reply:
Nvm I see your initial comment now. I'm sure it's being removed intentially.

Reply:
Yup it's intentional.